redisinsight/ui/src/components/query/query-card/QueryCard.tsx

```diff
@@ -196,50 +202,65 @@ const QueryCard = (props: Props) => {
               ? <QueryCardCommonResult loading={loading} result={commonError} />
               : (
                 <>
-                  {isGroupResults(resultsMode) && (
-                    <QueryCardCliResultWrapper
-                      loading={loading}
-                      query={command}
-                      db={db}
-                      resultsMode={resultsMode}
-                      result={result}
-                      isNotStored={isNotStored}
-                      isFullScreen={isFullScreen}
-                      data-testid="group-mode-card"
-                    />
-                  )}
-                  {(resultsMode === ResultsMode.Default || !resultsMode) && (
-                    <>
-                      {viewTypeSelected === WBQueryType.Plugin && (
-                        <>
-                          {!loading && result !== undefined ? (
-                            <QueryCardCliPlugin
-                              id={selectedViewValue}
-                              result={result}
-                              query={command}
-                              mode={mode}
-                              setMessage={setMessage}
-                              commandId={id}
-                            />
-                          ) : (
-                            <div className={styles.loading}>
-                              <EuiLoadingContent lines={5} data-testid="loading-content" />
-                            </div>
-                          )}
-                        </>
-                      )}
-                      {(viewTypeSelected === WBQueryType.Text) && (
-                        <QueryCardCliResultWrapper
-                          loading={loading}
-                          query={command}
-                          resultsMode={resultsMode}
-                          result={result}
-                          isNotStored={isNotStored}
-                          isFullScreen={isFullScreen}
-                        />
-                      )}
-                    </>
-                  )}
+                  {isSizeLimitExceededResponse(result)
```

Comment:
A lot of nesting 🥲

Reply:
Yeah. Otherwise we must split/refactor entire component which is time-consuming for such small task.

Reply:
If any of notices such deep nesting and ternaries being used in the rendering, or anything else that might be a suboptimal practice, but it is done to get it done for now. Please, tag me, so we can discuss it and make a task for it to improve it whenever we have time. Otherwise we'll forget about it and things like these or similar will keep falling through the cracks.
It will also be more productive than emotes ;) 
Tagging @pd-redis and @seth-riedel so they can see it as well

Reply:
why not use
```typescript
(isGroupResults(resultsMode) || isSizeLimitExceededResponse(result)) ? ...
```

Reply:
Love the initiative, @KIvanow! While I’m slightly skeptical about the 'whenever we have time' part, it’s definitely a step forward. And hey, emojis might be simple, but they’ve got some serious discussion-starting power! 😸✨